### PR TITLE
Finish building cell render method

### PR DESCRIPTION
**Pull request by lpile**
What does this PR do?
--
This PR has the `render` method created in `Cell` class which shows the different options available for the board.

I also had to fix some of the test.

### CODE REVIEW

File: test/cell_test.rb [hunk 1]
```diff
@@ -52,32 +52,28 @@ def test_fire_upon_updates_fired_upon_to_true
...
-
     assert_equal ".", @cell.render
   end
 
   def test_it_displays_an_s_if_it_has_a_ship_with_render_true_argument
     @cell.place_ship(@ship)
-    @cell.render(true)
-
-    assert_equal "S", @cell.render
+    assert_equal "S", @cell.render(true)
```
Comment (BrennanAyers) [inline test/cell_test.rb:70]: This looks great! My brain wasn't working yesterday, and this makes a lot more sense.

File: test/cell_test.rb [hunk 2]
```diff
@@ -52,32 +52,28 @@ def test_fire_upon_updates_fired_upon_to_true
   end
 
   def test_it_displays_a_dot_if_it_has_not_been_fired_upon
+    @cell.place_ship(@ship)
```
Comment (BrennanAyers) [inline test/cell_test.rb]: According to the spec, we should not need to have a Ship inside of the Cell to display a `dot`.

Comment (BrennanAyers) [reply test/cell_test.rb]: @lpile

**Review by BrennanAyers — CHANGES_REQUESTED** [review]
See comments @lpile

File: test/cell_test.rb (hunk 2 above)
Comment (lpile) [reply test/cell_test.rb]: I see, I will try something else

Comment (BrennanAyers) [reply test/cell_test.rb]:
```def render(boolean = false)
    if boolean == true
      "S"
    elsif @fire_upon == false
      "."```
Yes! This works! Great stuff!

**Review by BrennanAyers — APPROVED** [review]
I also like the change to "X"

**Pull request merged**